ts/torch_handler/image_classifier.py

```diff
@@ -70,7 +70,7 @@ def inference(self, data):
 
                 results.append(tmp)
             else:
-                results.append({str(classes[i]):str(probs[i])})
+                results.append({str(classes[index]):str(probs[index])})
```

Comment:
@TheRochVoices Thanks for this however it has been already taken care in [WIP] PR #432.